build.sbt

```diff
@@ -95,6 +95,70 @@ lazy val core = (project in file("core"))
     (compile in Compile) := ((compile in Compile) dependsOn createTargetClassesDir).value
   )
 
+lazy val deltacontribs = (project in file("delta-contribs"))
+  .enablePlugins(GenJavadocPlugin, JavaUnidocPlugin, ScalaUnidocPlugin)
+  .dependsOn(core)
```

Comment:
I think your tests are failing because the tests of this module is not depending on the tests of the core module. Hence the tests are showing this following compilation error
```
[info] Resolving org.antlr#antlr-master;3.5.2 ...
[error] /home/circleci/project/delta-contribs/src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala:21: not found: type LogStoreSuiteBase
[error] class COSLogStoreSuite extends LogStoreSuiteBase {
[error]                                ^
[error] /home/circleci/project/delta-contribs/src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala:24: value sparkConf is not a member of AnyRef
[error]     super.sparkConf.set(logStoreClassConfKey, logStoreClassName)
[error]           ^
[error] /home/circleci/project/delta-contribs/src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala:24: not found: value logStoreClassConfKey
[error]     super.sparkConf.set(logStoreClassConfKey, logStoreClassName)
[error]                         ^
[info] Resolving org.sonatype.oss#oss-parent;9 ...

[info] Resolving org.antlr#ST4;4.0.8 ...

[info] Resolving org.antlr#ST4;4.0.8 ...
[error] /home/circleci/project/delta-contribs/src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala:30: not found: value testHadoopConf
[error]   testHadoopConf(
[error]   ^
[error] /home/circleci/project/delta-contribs/src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala:31: not found: value expectedErrMsg
[error]     expectedErrMsg = ".*No FileSystem for scheme.*fake.*",
[error]     ^
[info] Resolving org.sonatype.oss#oss-parent;9 ...

[info] Resolving org.abego.treelayout#org.abego.treelayout.core;1.0.3 ...

[info] Resolving org.abego.treelayout#org.abego.treelayout.core;1.0.3 ...

[info] Resolving org.sonatype.oss#oss-parent;7 ...

[info] Resolving org.sonatype.oss#oss-parent;7 ...
[error] /home/circleci/project/delta-contribs/src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala:32: not found: type FakeFileSystem
[error]     "fs.fake.impl" -> classOf[FakeFileSystem].getName,
[error]                               ^
[error] 6 errors found
```

The fix probably is 
```
.dependsOn(core % "compile->compile;test->test")
```